diff -ur orinoco-0.15rc2.orig/orinoco_pci.c orinoco-0.15rc2/orinoco_pci.c
--- orinoco-0.15rc2.orig/orinoco_pci.c	2004-07-28 08:06:45.000000000 +0200
+++ orinoco-0.15rc2/orinoco_pci.c	2005-04-02 19:35:11.000000000 +0200
@@ -327,7 +327,7 @@
 	
 	orinoco_unlock(priv, &flags);
 
-	pci_save_state(pdev, card->pci_state);
+	pci_save_state(pdev);
 	pci_set_power_state(pdev, 3);
 
 	return 0;
@@ -344,7 +344,7 @@
 	printk(KERN_DEBUG "%s: Orinoco-PCI waking up\n", dev->name);
 
 	pci_set_power_state(pdev, 0);
-	pci_restore_state(pdev, card->pci_state);
+	pci_restore_state(pdev);
 
 	err = orinoco_reinit_firmware(dev);
 	if (err) {
